src/components/sidebar/components/sidebar-headings/sidebar-headings.module.css

```diff
@@ -1,10 +1,10 @@
 .root {
-  color: var(--token-color-foreground-primary);
-  font-size: var(--token-typography-display-100-font-size);
-  font-weight: var(--token-typography-font-weight-semibold);
-  line-height: var(--token-typography-display-100-line-height);
-  margin-bottom: 12px;
-  margin-top: 0;
-  padding-left: 8px;
-  padding-right: 8px;
+	color: var(--token-color-foreground-primary);
+	font-size: var(--token-typography-display-100-font-size);
+	font-weight: var(--token-typography-font-weight-semibold);
+	line-height: var(--token-typography-display-100-line-height);
+	margin-bottom: 12px;
+	margin-top: 20px;
```

Comment:
Is there a particular issue being solved with this setting? It causes the spacing to not match design spec when there is a heading item directly after the filter input. ([Figma link](https://www.figma.com/file/VD7ahvXuXWJApeGnhbW4hv/Developer?node-id=11303%3A116341))

<img width="883" alt="CleanShot 2022-08-15 at 16 48 17@2x" src="https://user-images.githubusercontent.com/43934258/184715747-2d7a2feb-613e-491b-bbb6-bf852dfe6739.png">

Reply:
Is this one of the pages that the change was needed? https://dev-portal-git-zsfix-terraform-sidebar-overview-item-hashicorp.vercel.app/terraform/plugin

Reply:
☝️ yep, exactly - it's where headings appear in the midst of nav-data. Also I am totally face palming right now because I wrote review comments in a text file and then forgot to post them 🤦

Reply:
Update: I've tweaked this to `16px` since we do have a design spec for a heading at the top of nav data, but not for one in middle of nav data. Thanks for catching this & linking the spec 💖

Reply:
That should work well! 👍🏻 It'll also continue working if/when dividers are added in since those have a larger margin on them.